src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -176,6 +195,7 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		});
 
 		this._ctxMenuWidgetVisible = Context.Visible.bindTo(this._contextKeyService);
+		this.listRenderer = new CodeMenuRenderer([`onEnterSelectCodeAction`, `onEnterSelectCodeActionWithPreview`], keybindingService);
```

Comment:
Instead of passing these in, can you extract these ids to some common file that both this code and the command registration itself then use?